packages/web-api/src/controllers/scan-request-controller.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Context } from '@azure/functions';
+import { Guid, RestApiConfig, ServiceConfiguration, Url } from 'common';
+import { inject, injectable } from 'inversify';
+import { Logger } from 'logger';
+import { ScanRunRequest } from '../api-contracts/scan-run-request';
+import { ScanRunResponse } from '../api-contracts/scan-run-response';
+import { ScanDataProvider } from '../providers/scan-data-provider';
+import { webApiIocTypes } from '../setupIoContainer';
+import { ApiController } from './api-controller';
+
+@injectable()
+export class ScanRequestController extends ApiController {
+    public readonly apiVersion = '1.0';
+    public readonly apiName = 'web-api-post-scans';
+
+    public constructor(
+        @inject(webApiIocTypes.azureFunctionContext) protected readonly context: Context,
+        @inject(ScanDataProvider) private readonly scanDataProvider: ScanDataProvider,
+        @inject(ServiceConfiguration) private readonly serviceConfig: ServiceConfiguration,
+        @inject(Logger) protected readonly logger: Logger,
+    ) {
+        super();
+    }
+
+    public async handleRequest(): Promise<void> {
+        const payload = this.tryGetPayload<ScanRunRequest[]>();
+        if (payload === undefined) {
+            return;
+        }
+
+        const maxLength = (await this.getRestApiConfig()).maxScanRequestBatchCount;
```

Comment:
>     const maxLength = (await this.getRestApiConfig()).maxScanRequestBatchCount; [](start = 3, length = 80)

we are not copying service config in azure function scenario. added user story for this.

Reply:
ok